### ISSUE TITLE
[Bug]: Local installation not able to complete

### ISSUE THREAD

**Issue by neonJAhr** [jasp-stats/jasp-issues#2841]
### JASP Version

0.19.0

### JASP Module

Unrelated

### What OS are you seeing the problem on?

Windows 10

### Bug Description

Hey JASP team, 

I was installing version 0.19 with the .msi executable. Under the "Advanced" option I selected a local account rather than system-wide installation. However, midway through I got this error message. 
![error](https://github.com/user-attachments/assets/41911152-de86-46c7-a60e-142cad84f2b7 "Error Message") 
My Windows version is 10.0.19045, I can happily post more system info. 
Also, a system-wide installation worked fine. I was able to reproduce this error with and without JASP 0.18.3 pre-installed

### Expected Behaviour

This installation should have run.

### Steps to Reproduce

1. Download & run "JASP-0.19.0.0-Windows.msi"
2. Select Installation Scope, click Advanced
3. On Product Features, for Visual C++ 14.0 Runtime select "Entire Feature will be installed on local hard drive". Note that the installer says 0KB will be required.
4. On pop-up, select OK

![image](https://github.com/user-attachments/assets/912d3e6e-8f40-44fb-87d7-5646a35aadcc "Additional Installer Window")

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by tomtomme** [comment]
Can you install via tha Microsoft store maybe? 
Also I think in an earlier step the "normal" installer you used, has an "advanced" option where you can select between a system wide or user only installation. The latter should install without admin privileges

**Comment by tomtomme** [comment]
Thx for the details. So local install is affected only. System wide works. 

@RensDofferhoff 
We should definitely fix this one, right? Or will the msi installer be discontinued soon?
I guess the msix installer via ms-store does not have this "local" vs. "system-wide" feature, right?

**Comment by RensDofferhoff** [comment]
@neonJAhr Thanks for the report! 
I might be able to fix the msi.
But as @tomtomme  mentioned I would really like to get rid of that arcane installer type.
In the meantime you could consider the Windows Store install. These should be installed on user level.
If that is not to your liking we also have the ZIP which can just be extracted and run without system install https://jasp-stats.org/thank-you-for-downloading-jasp-win64-zip/

**Comment by shun2wang** [comment]
Oh, may bad. that was what I made for disable ms-gamingoverlay things for .msi, so it may because some registry should not work without administrator permission. but that still weird because we(me and user) both tested .msi build it worked good...

so remove this or try make it only work for administrator:
```xml
			<RegistryKey Root="HKMU" Key="Software\Policies\Microsoft\Windows\GameDVR">
				<RegistryValue Name="AllowGameDVR" Value="0"   Type="integer"  />
			</RegistryKey>
```

see: https://github.com/jasp-stats/jasp-issues/issues/2479 and https://github.com/jasp-stats/jasp-desktop/pull/5405

**Comment by Jilgamesh91** [comment]
Hello,
I have the same problem. I downloaded the zip file and activated installation. I need to install Jasp for user only because otherwise I would need admin permissions, but before the installation ends, It appears the pop-up above and then a series of pop-ups telling me almost every module was not installed. So, JASP becomes unasable. Sorry, but I can't install it via Microsoft Store. Is there a solution?

Thanks

**Comment by shun2wang** [comment]
hello all,

could you try this version and see if it still hhappen. https://static.jasp-stats.org/Nightlies/Windows/JASP-nightly-gameDVR-fcd2ad6f84cb12ee26db6e3236462efdef1d5ae0.msi

**Comment by RensDofferhoff** [comment]
I just tried this. The install doesnt fail now but the wix stuff that sets the modules doesnt work if the folder is not the same as machine wide install.



@Jilgamesh91 Could you give more info as to why the ZIP does not work?

**Comment by shun2wang** [comment]
>but the wix stuff that sets the module junctions doesnt work if the folder is not the same as machine wide install.

Sorry I am not sure if I understand this...we didn't touch module junctions anyway during 0.19 developing.

**Comment by RensDofferhoff** [comment]
No I think this just never worked for user level install

**Comment by RensDofferhoff** [comment]
@Jilgamesh91 Could you try this ? https://static.jasp-stats.org/JASP-0.19.0.msix

**Comment by Jilgamesh91** [comment]
Hello, I tried both the link you gave me, but the downloads failed (see the error attached)
<img width="275" alt="Screenshot 2024-08-27 122318" src="https://github.com/user-attachments/assets/999e38f2-fd86-4f71-a267-94ef9624e7e3">

**Comment by RensDofferhoff** [comment]
@Jilgamesh91 And the ZIP option also didnt work?
There are a lot of restrictions on your PC.

**Comment by Jilgamesh91** [comment]
@RensDofferhoff Hi, with ZIP option do you mean the one in the screenshot below? If yes, I tried it and doesn't work, you can only install on user and then doesn't install all the modules
<img width="845" alt="Screenshot 2024-08-27 152147" src="https://github.com/user-attachments/assets/0c7dcfb1-9c0c-4986-bad4-48e6ae03a11f">

**Comment by RensDofferhoff** [comment]
@Jilgamesh91 No just below that is a download link to the zip:
https://jasp-stats.org/thank-you-for-downloading-jasp-win64-zip/

You can extract this anywhere and run the jasp executable in that folder.

**Comment by Jilgamesh91** [comment]
It works now! Thank you very much!

**Comment by RensDofferhoff** [comment]
Perhaps we should advertise it more clearly on the website. @boutinb what are your thoughts on this? Its rather hidden now.

**Comment by boutinb** [comment]
That's true that you have to read really carefully the page to see the zip version. We should change this. Tomorrow we can do this.

**Comment by tomtomme** [comment]
The actuall fix will be available in 0.19.2, commit here: https://github.com/jasp-stats/jasp-desktop/pull/5639
